agent/dns.go

```diff
@@ -620,7 +628,7 @@ func encodeKVasRFC1464(key, value string) (txt string) {
 }
 
 // formatNodeRecord takes a Node and returns an A, AAAA, TXT or CNAME record
```

Comment:
nit: I'd update the doc comment to describe the semantics of the two returned slices.